h/emails/reply_notification.py

```diff
@@ -20,11 +21,8 @@ def generate(request, notification):
     if not document_title:
         document_title = notification.parent.target_uri
 
-    parent_url = request.route_url('annotation', id=notification.parent.id)
-    reply_url = request.route_url('annotation', id=notification.reply.id)
-
-    reply_user_url = request.route_url('stream.user_query',
-                                       user=notification.reply_user.username)
+    reply_url = (links.incontext_link(request, notification.reply) or
```

Comment:
IMO it would be clearer and more consistent with, for example, the code immediately above if this was written out explicitly:

```python
reply_url = links.incontext_link(request, notification.reply)
if not reply_url:
    reply_url = request.route_url('annotation', id=notification.reply.id)
```